src/middlewares/error-handler.ts

```diff
@@ -0,0 +1,8 @@
+import { NextFunction, Request, Response } from 'express';
+export const ErrorHandler = (err: any, req: Request, res: Response, next: NextFunction) => {
```

Comment:
```suggestion
import { NextFunction, Request, Response } from 'express';

export const ErrorHandler = (err: any, req: Request, res: Response, next: NextFunction) => {
```
Опять вопрос где пропуск строчки, и почему он сам не выставляется?